tests/e2e/test_rosacli_ingress.go

```diff
@@ -145,6 +145,23 @@ var _ = Describe("Edit default ingress",
 						"ERR: Updating route selectors is not supported for Hosted Control Plane clusters"))
 			})
 
+		It("can describe ingress of a cluster - [id:73538]",
+			labels.Low, labels.Runtime.Day2,
+			func() {
+				By("Retrieve cluster and get default ingress id")
+				output, err := ingressService.ListIngress(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+
+				ingressList, err := ingressService.ReflectIngressList(output)
+				Expect(err).ToNot(HaveOccurred())
+				var defaultID string
+				for _, v := range ingressList.Ingresses {
```

Comment:
You can write the code here like this

If len(ingressList.Ingresses) > 0 {
          defaultID = ingressList.Ingresses[0]
 else {
         log.logger.Error("No default ingress is present to the cluster")
}